src/_guides/language/index.md

```diff
@@ -40,7 +40,7 @@ These two resources are popular with both beginning Dartisans and experts.
 <li markdown="1">Asynchronous programming
 
 <ul markdown="1">
-<li markdown="1">[Futures](/tutorials/language/futures)<br>
+<li markdown="1">[Asynchrony: futures, async, await](/codelabs/async-await)<br>
     A beginner's guide to handling asynchronous programming in Dart.
```

Comment:
The description we have in the codelab is:
`Learn about and practice writing asynchronous code in DartPad!`

Here we're describing it with:
`A beginner's guide to handling asynchronous programming in Dart.`

Maybe having different descriptions for the codelab is a good thing, just thought I'd mention it.

Reply:
I think `A beginner's guide` leaves out the DartPad / interactivity of the codelab, where other descriptions have captured that. I think `Learn about and practice writing asynchronous code in DartPad!` works here.

Reply:
![Screen Shot 2019-08-12 at 11 50 33 AM](https://user-images.githubusercontent.com/5550175/62894596-dce31f80-bd01-11e9-86a5-03d3608688d2.png)

Nesting "Asynchrony" beneath "Asynchronous Programming" can be a little jarring, since "Asynchrony" is more abstract than "Asynchronous Programming". 

Another option might be: 

Change the title of the codelab from "Asynchrony: ..." to "Asynchronous programming: ...", which is what we had for the tutorial and what we're currently doing for the streams tutorial. If so, we could make this section look something like this:

```
Asynchronous Programming:
- futures, async, await
- streams
```


(2) Change the link text to "asynchrony codelab" like we are doing below.

Reply:
> IMHO `Learn about and practice writing asynchronous code in DartPad!` works here.

Exclamation points don't work anywhere in our docs. :) https://developers.google.com/style/exclamation-points

(I think they're more OK in social sharing text. Possibly OK in codelabs. Not OK anywhere else.)